src/App.jsx

```diff
@@ -3,13 +3,15 @@ import Hotels from './pages/Hotels/Hotels';
 import Rooms from './pages/Rooms/Rooms';
 import Home from './pages/Home/Home';
 import UserRegistration from './pages/UserRegistration/UserRegistration';
+import HotelsRegistration from './pages/HotelsRegistration/HotelsRegistration';
 
 const App = () => (
   <Routes>
     <Route path="/" element={<Home />} />
     <Route path="/hotels" element={<Hotels />} />
     <Route path="/hotel/:id" element={<Rooms />} />
     <Route path="/register" element={<UserRegistration />} />
+    <Route path="/hotelsRegistration" element={<HotelsRegistration />} />
```

Comment:
Lo más recomendable para el path es seguir los patrones slug: 

https://boldist.co/search-engine-marketing/how-to-write-a-slug/